index.html

```diff
@@ -122,6 +177,196 @@ <h2>Methods</h2>
       </dl>
     </section>
   </section>
+  <section id="rtcrtptransceiver-interface">
+    <h3>
+      <dfn>RTCRtpTransceiver</dfn> extensions
+    </h3>
+    <p>
+      The <code><a href="https://w3c.github.io/webrtc-pc/#dom-rtcrtptransceiver">RTCRtpTransceiver</a></code>
+      interface is defined in [[WEBRTC]]. This document extends that interface
+      by adding an additional method and attribute in order to control negotiation of
+      RTP header extensions.
+    </p>
+    <pre class="idl">
+partial dictionary RTCRtpHeaderExtensionCapability {
+  RTCRtpTransceiverDirection direction = "sendrecv";
+};
+      
+partial interface RTCRtpTransceiver {
+  void setOfferedRtpHeaderExtensions(
+sequence&lt;RTCRtpHeaderExtensionCapability&gt; headerExtensionsToOffer);
+  readonly attribute FrozenArray&lt;RTCRtpHeaderExtensionCapability&gt; headerExtensionsToOffer;
+  readonly attribute FrozenArray&lt;RTCRtpHeaderExtensionCapability&gt; headerExtensionsNegotiated;
+};
+    </pre>
+    <p>
+      Let [[\HeaderExtensionsToOffer]] be an internal slot of the RTCRtpTransceiver,
+      initialized to the platform-specific list of implemented RTP header extensions.
+      The direction
+      attribute for all extensions that are mandatory to use MUST be initialized to
+      an appropriate value other than "stopped". The direction attribute for extensions
```

Comment:
I'm not clear on whether getCapabilities() should return direction at all; it is able to do so because we used "partial dictionary", but that doesn't mean that we have to.
I think the Principle of Least Astonishment says that sender.getCapabilities() should return the same thing as "addTrancsceiver().headerExtensionsToOffer()", filtered to remove header extensions that can't be set on a sender (if any). Returning differing values seems like a gotcha that we don't need to make.